.ci/azure-pipelines/azure-pipelines.yaml

```diff
@@ -92,6 +93,7 @@ stages:
         variables:
           BUILD_DIR: '$(Agent.BuildDirectory)/build'
           CMAKE_CXX_FLAGS: '-Wall -Wextra'
+          DISPLAY: :99.0
```

Comment:
This comment is just a commentary on how weird it's that DISPLAY can't be used with the xvbf call without serious paramter substitution tricks